exo-setup/src/app-setup.ls

```diff
@@ -27,15 +27,16 @@ class AppSetup extends EventEmitter
         @services.push do
             role: service-role
             location: service-data.location
-            docker-image: service-data.docker_image
+            docker-image: service-data['docker-image']
 
     @_setup-services ~>
-      @_get-dependencies-docker-config!
-      @_get-service-docker-config!
-      @_render-docker-compose!
-      @_setup-docker-images (exit-code) ~>
-        | exit-code => @write 'setup failed'; process.exit exit-code
-        @write 'setup complete'
+      @_get-dependencies-docker-config (err) ~>
+        | err => @write 'setup failed'; process.exit
```

Comment:
Will update to exit with code 1. This is part of the larger setup process, which is tested in `setup.feature`,  so if this step exits then the setup process won't finish and those tests won't pass.

Although when I implemented the docker-compose stuff I did add a test diff-ing the `docker-compose.yml` file which was created, which included checking the Docker config set in this function `@_get-dependencies-docker-config`. @kevgo made the point that since the file is always auto-generated it doesn't concern the user and shouldn't be in a feature test. Thoughts on adding that back in? Or maybe unit testing that instead?